clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -809,10 +809,6 @@ scheduler_tmpl_t<RecordType, ReaderType>::set_initial_schedule(
     // The filetype, if present, is before the first timestamp.  If we only need the
     // filetype we avoid going as far as the timestamp.
     bool gather_filetype = options_.read_inputs_in_init;
-    // Avoid reading ahead for replay as it makes the input ords not match in tests.
```

Comment:
Trying to remember why we had this in the first place. The change in clients/drcachesim/tests/scheduler_unit_tests.cpp seems simple enough, but did we not want to make it before because it's not really correct (should be 1 and not 2)?

Reply:
Yes that single unit test seems to be the sole reason.  I think at the time I didn't realize there weren't more tests beyond that and thought more test tweaks would be needed.  Also note that the ordinal is already ahead for files passed in because open_reader() reads ahead already so this is not really a change for normal upstream uses.